comparisontool/management/commands/load_school.py

```diff
@@ -11,33 +13,39 @@ class Command(BaseCommand):
     args = '<path>'
     help = 'Load the specified CSV into the schools database'
 
+    def add_arguments(self, parser):
+        parser.add_argument('path_spec')
+
     def handle(self, *args, **options):
-        for path_spec in args:
-            for path in glob(path_spec):
-                print "loading %s" % path
-                with open(path) as csv_file:
-                    reader = csv.DictReader(csv_file)
-                    for record in reader:
-                        # Last load took 14 minutes, this will add a visual indicator that smth is happening
-                        print record['SCHOOL_ID']
-                        aliases = set()
-                        primary_school_name = record.get("SCHOOL")
-                        aliases.add(primary_school_name)
-                        aliases_packed = record.get('ALIAS')  # pipe delimited
-                        if aliases_packed:
-                            aliases = aliases.union(aliases_packed.split('|'))
-                        city = record.get('CITY')
-                        state = record.get('STATE')
-                        school_id = int(record['SCHOOL_ID'])
-                        data_json = dumps(record)
-                        school = School(school_id=school_id,
-                                        city=city,
-                                        state=state,
-                                        data_json=data_json)
-                        school.save()
-                        school.alias_set.all().delete()
-                        for (index, alias_str) in enumerate(aliases):
-                            alias = Alias(institution=school,
-                                          alias=alias_str,
-                                          is_primary=(alias_str == primary_school_name))
-                            alias.save()
+        path_spec = options['path_spec']
+        for path in glob(path_spec):
+            print("loading %s" % path)
+            with open(path) as csv_file:
+                reader = csv.DictReader(csv_file)
+                print("Schools loaded: ", end="")
+                for record in reader:
+                    # Last load took 14 minutes, this will add a visual indicator
+                    #  that something is happening
+                    print(record['SCHOOL_ID'], end=", ")
```

Comment:
Happy to revert these `print` changes if they're too silly :)

Reply:
These are definitely not too silly -- having output is great, and helpful for the user.

This is out of scope for this PR, but it would be nice to suppress these message during unit testing. Generally in other projects we've tried to avoid printing to the console during tests; it clutters up the nice `.....` output. At one point @willbarton even opened [a PR](https://github.com/cfpb/cfgov-refresh/pull/3800) to enforce this in cfgov-refresh.

Reply:
Good call. I created ticket #10 to remind us.

Reply:
> At one point @willbarton even opened a PR to enforce this in cfgov-refresh.

Would it be useful if the `StdoutCapturingTestRunner` was a separate library from cfgov-refresh? Then we could use it independently in other apps, like this one.